docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,73 @@
+<p style="display:inline;">You can find this graph in the Monitor tab of an experiment. Plug in the conversion rates and sample size in <a href="https://www.socscistatistics.com/tests/ztest/default2.aspx">here</a> and see if its statistically significant or not at the 95% significance level. The conversion rates between variants should be similar (within randomness) because the assignment event should randomly split users into two equal cohorts. Since exposure event gets sent right before the user experiences a variant, there should be no difference between users since they have the same user experience from the time the assignment event is sent ot the time the exposure event is sent.</p>
+<p></p>
+#### 6. Did you add/remove a variant in the middle of an experiment?
+<p style="display:inline;">This is just a special case of #2.</p>
+<p></p>
+#### 7. Was sticky bucketing turned on and off in the middle of the experiment?
+<p style="display:inline;">Generally, once you start an experiment you should not change anything.</p>
+<p></p>
+#### 8. Are there a lot of users in the ["Individual Users Allocation"](../general/evaluation/implementation.md#Individual-inclusions) section?
+<p style="display:inline;">This can cause an SRM because you are not randomly allocating those users. It is fine to add some users in the experiment in the "Individual Users Allocation".</p>
+<p></p>
+#### 9. Is this your first experiment?
+<p style="display:inline;">Maybe there is an instrumentation bug. If you have had multiple SRMs, then the SRM is unlikely to be a false positive.</p>
+<p></p>
+#### 10. Are users logging in and out a lot?
+<p style="display:inline;">You may see this manifested in the variant jumping chart. Is the experiment causing users to log out more? This happens for a lot of financial institutions where you get logged out of your bank account if you are inactive for more than 15 minutes.</p>
+
+>    1. On log out regenerate device id<br>
+>    2. Amplitude thinks its a new user<br>
+>    3. User can get bucketed into a different variant<br>
+>    4. Then id gets resolved<br>
+
+This can also be related to anonymous users from sign up flow experiments. 
+<p></p>
+#### 11. Are there two users who have the same sequence of events but one is missing exposure?
+<p style="display:inline;">Maybe there are some cases where you are not sending the exposure event or are spending the exposure event when you should not be.</p>
+<p></p>
+#### 12. If the customer is not using the Amplitude SDK is the fall back variant causing issue?
+<p style="display:inline;">Is there some case where if the flag does not return a response in a reasonable amount of time, the flag default to control and counts that as an exposure? If so, control will have more exposures than treatment.</p>
+
+!!! info
+    Our client side sdk doesn’t send exposure events for fallback variant
```

Comment:
"client side sdk doesn’t send exposure events for fallback variant" --> "client-side SDKs don't send exposure events for fallback variants"